.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -17,5 +17,6 @@ about: Suggest an idea for this project
 **Environment:**
 
 - Cluster-api version:
+- Minikube version:
```

Comment:
I'm wondering if we even need the Environment section for a feature request, it seems like it is a bit extraneous.

Reply:
I also have the same concern as you when update here, let me delete this.